<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OUTItemWidget</class>
 <widget class="QWidget" name="OUTItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>434</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Type d'Item :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_itemType"/>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Alias d'indexation :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="alias"/>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Nom du mod√®le d'item :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="modelName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Description de l'item :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="modelDescription">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
